DockerizeJenkinsfile

```diff
@@ -7,7 +7,7 @@ def use_release_artifacts() {
 pipeline {
     agent {
         docker {
-            image 'registry.gear.ge.com/dig-predix-security-services/uaa-ci-testing:1.1.0'
+            image 'registry.gear.ge.com/dig-predix-security-services/uaa-ci-testing:latest'
```

Comment:
Any reason for doing this other than just to use the latest? Would be nice to also mention this change in the commit message. So that if this somehow causes problems in the future, it will help to know whether this was actually needed for the UAA to be tested correctly or something.

Reply:
uaa-ci-testing is the image of UAA testing environment that we build and maintain ourselves.  It's used across all our UAA pipelines.  The initial version 1.1.0 supports Java 8.  The latest version supports Java 11.

Reply:
Updated commit message.